code/src/components/button.css

```diff
@@ -0,0 +1,20 @@
+button {
+	margin: 20px;
+	background: transparent;
+	color: white;
+	font-size: 20px;
+	font-weight: bold;
+	border-style: none;
+	display: flex;
+	align-items: center;
+}
```

Comment:
Think about indentation ☺️

Reply:
Fooork... I can't get the indentation to show up nice in github, it looks great in VSCode...